packages/expo-updates/android/src/main/java/expo/modules/updates/loader/FileDownloader.kt

```diff
@@ -25,8 +26,16 @@ import org.apache.commons.fileupload.MultipartStream
 import org.apache.commons.fileupload.ParameterParser
 import java.io.ByteArrayOutputStream
 
-open class FileDownloader(context: Context) {
-  private val client = OkHttpClient.Builder().cache(getCache(context)).build()
+open class FileDownloader() {
+  private lateinit var client: OkHttpClient
+
+  constructor(context: Context) : this() {
+    client = OkHttpClient.Builder().cache(getCache(context)).build()
```

Comment:
better to call:
```
this(OkHttpClient.Builder().cache(getCache(context)).build())
```
?

then `constructor(client: OkHttpClient)` can become the primary constructor, and `lateinit var client` becomes `val client` again.